sources/iTermKeyBindingMgr.m

```diff
@@ -471,6 +471,22 @@ + (NSString *)formatAction:(NSDictionary *)keyInfo
                             [self stringForSelectionMovementUnit:auxText.integerValue]];
             break;
 
+        case KEY_ACTION_DECREASE_ROW:
+            actionString = @"Decrease Row";
```

Comment:
Please rename the actions, constants, and methods to use this wording:

 * Decrease Height
 * Increase Height
 * Decrease Width
 * Increase Width